### ISSUE TITLE
Bounds for agent position/rotation/horizon

### ISSUE THREAD

**Issue by rkruegs123** [allenai/ai2thor#255]
What are the bounds for x/y/z in position and rotation, and for horizon?

**Comment by roozbehm** [comment]
Please refer to the [documentation](http://ai2thor.allenai.org/documentation/actions/navigation) for navigation actions to see how to get the coordinates for navigable points and rotation and horizon values.

**Comment by rkruegs123** [comment]
I understand how to use the navigation actions and how to retrieve the coordinates. My question pertains to the bounds of the position and rotation vectors given by event.metadata['agent']['position'] and event.metadata['agent']['rotation']. What are the minimum and maximum values for x/y/z in both vectors?

**Comment by roozbehm** [comment]
Then, it is just simply finding the minimum and maximum x in the list of points. Or are you looking for something else?

**Comment by rkruegs123** [comment]
Given an event, it is not just that I am interested in the current min/max. I am interested in the min/max for every possible point.

In simpler terms, one of my questions would be: What is the range of possible values for event.metadata['agent']['position']['x']? Is there any better answer for this aside from just using the reachable positions and getting a minimum and maximum manually?

**Comment by roozbehm** [comment]
No, we do not have a function for that.

**Comment by thomason-jesse** [comment]
It looks like [sceneBounds](https://ai2thor.allenai.org/ithor/documentation/metadata/#event-attributes) in the newer release gives this information.
